.nvmrc

```diff
@@ -1 +1 @@
-v16.17.0
+v16.17.1
```

Comment:
@KevinGhadyani-Okta  We need to fix the build script as well 

https://github.com/okta/odyssey/blob/develop/scripts/setup.sh#L3